tasks/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ add_dependencies(${ROCK_GAZEBO_TASKLIB_NAME}
 
 TARGET_LINK_LIBRARIES(${ROCK_GAZEBO_TASKLIB_NAME}
     ${OrocosRTT_LIBRARIES}
-    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES})
+    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES}
+    thruster_types)
```

Comment:
I've created a google protobuf msg library in **simulation/gazebo_thruster**. 
We use this msg to send/receive data from gazebo topic and we use this lib here.
You may be willing to rename it... **thruster_msg** !?